src/CribblyBackend/Startup.cs

```diff
@@ -52,7 +51,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IMigrati
             {
                 endpoints.MapControllers();
             });
-            migrationRunner.MigrateUp();
+            if (!env.IsDevelopment())
+            {
+                var runner = app.ApplicationServices.GetService<IMigrationRunner>();
+                runner.MigrateUp();
+            }
```

Comment:
Can't remember why I did this. I guess it was because the migrator was trying to run in unit tests? In any case, if you run with docker, it'll run the release build, so the migrations should run.